src/config/setenv/setenv_dev.ini

```diff
@@ -11,4 +11,5 @@ export FCDB_DB_POOL_SIZE=1
 export FCDB_DB_POOL_OVERFLOW=5
 
 export FCDB_HOST=localhost
-export FCDB_PORT=5432
+#export FCDB_PORT=5432
+export FCDB_PORT=6630
```

Comment:
I guess this change is left from your testing setup and should not be added to the change